core/server/src/main/java/alluxio/worker/AlluxioWorker.java

```diff
@@ -38,13 +38,16 @@ public static void main(String[] args) {
       System.exit(-1);
     }
 
-    AlluxioWorkerService worker = new DefaultAlluxioWorker();
+    AlluxioWorkerService worker = null;
     try {
+      worker = new DefaultAlluxioWorker();
```

Comment:
maybe instantiate this in its own try-catch block. We don't want to call `stop` if the call to `start` failed